packages/common/package.json

```diff
@@ -8,8 +8,11 @@
     "test": "jest"
   },
   "dependencies": {
-    "@ckb-lumos/lumos": "0.20.0",
     "env-paths": "2.2.1",
     "undici": "5.22.1"
+  },
+  "devDependencies": {
+    "@ckb-lumos/base": "0.20.0",
+    "@ckb-lumos/lumos": "0.20.0"
```

Comment:
Why to move these two dependencies to `devDependencies`?

Reply:
They are only referenced in https://github.com/ckb-js/kuai/blob/develop/packages/common/src/util.ts#L1-L2 as type definitions and will be wiped out after compilation. In other words, they are not used for production.

Reply:
> I think it may not only be referenced for type definition in the future.

They can be listed in `dependencies` when used for production.
Listing unused packages in `dependencies` because they may be used in the future is somehow over-optimized.

Reply:
I found this conversation has been marked solved, does that mean PR has been reviewed?

Reply:
Yes, and I reviewed other dependencies, and no other missing found.